rel/config.exs

```diff
@@ -19,17 +19,13 @@ environment :prod do
 end
 
 release :elixir_node do
-  set version: "0.1.0"
+  set version: System.get_env("VERSION")
```

Comment:
what's the purpose of this?

Reply:
to set version number as environment while releasing, to not have to change it manually in this file

Reply:
got it, thanks